config.js

```diff
@@ -123,6 +123,8 @@ module.exports =
 , newrelic              : { key: process.env.NEW_RELIC_LICENSE_KEY
                                                                 || false }
 , wallet                : { }
+
+, testingCohorts        : process.env.TESTING_COHORTS ? process.env.TESTING_COHORTS.split(',') : []
```

Comment:
i suspect that the better idiom is

        (process.env.TESTING_COHORTS && process.env.TESTING_COHORTS.split(',')) || []